### DISCUSSION TITLE
Apollo materials

### DISCUSSION THREAD

**Discussion by neurvanna** [Coen-Lab/chronic-neuropixels#15] [General]
Hello! Is there any specific reason to make some parts from Natural Rigid Resin 4000 while others from PA12, and what would happen if everything was made from PA12? Thank you ☺️

**Comment by pipcoen** [comment]
Hey Anna,
In short, we used Natural Rigid Resin 4000 because it was stronger, and PA12 because it was lighter. I think it would be fine (and lighter) to make everything from PA12. Especially since once it's glued together, it's pretty sturdy. However, we haven't actually tried implanting a complete PA12 implant (although we printed one, and it seemed okay). You could also increase the wall-thickness parameter a bit to make it sturdier. 

If you do decide to go down this route, keep us updated on how it goes!
